cmd/airflow.go

```diff
@@ -189,6 +191,7 @@ func newAirflowUpgradeTestCmd(astroClient astro.Client) *cobra.Command {
 	cmd.Flags().BoolVarP(&dagTest, "dag-test", "d", false, "Only run DAG tests. These tests check whether your DAGs will generate import errors after you upgrade.")
 	cmd.Flags().StringVarP(&deploymentID, "deployment-id", "i", "", "ID of the Deployment you want run dependency tests against.")
 	cmd.Flags().StringVarP(&customImageName, "image-name", "n", "", "Name of the upgraded image. Updates the FROM line in your Dockerfile to pull this image for the upgrade.")
+	cmd.Flags().StringSliceVar(&buildSecrets, "build-secrets", []string{}, "Mimics docker build --secret flag. See https://docs.docker.com/build/building/secrets/ for more information. Example input id=mysecret,src=secrets.txt")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&buildSecrets, "build-secrets", []string{}, "Expose a secret to containers. Equivalent to 'docker build --secret'. Example input id=mysecret,src=secrets.txt")
```
I would remove the link from this help text, both because the link could break and the user has to manually highlight and paste it

Reply:
the input is pretty specific to `docker build --secret`. Really too much to explain in the help text. Should we just tell users tot look this information up. id and src in the example are secret types that dockers command takes. you must use a docker secret type with this flag. a list of secret types can be found in the link

Reply:
Updated my suggestion, I think the "equivalent to" here makes it clear that this is a mimic, and that we won't be explaining the entire usage. Just wanted to avoid including a raw URL in help text to external docs

Reply:
Also we should make this update to all commands where this help text is used